docs/how-to-contribute/general-onboarding.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 3
+title: "General Onboarding Checklist"
+sidebar_label: "General Onboarding Checklist"
+---
+# Step 1  General Setup
+- [ ] Sign up for [Discord](https://discord.gg/RFufzsqRmJ) and introduce yourself to the community
+- [ ] Sign up for [openbeta.io](https://openbeta.io/)
+- [ ] Subscribe to [our newsletter](https://docs.openbeta.io/stay-connected)
+- [ ] Connect with us on [Instagram](https://www.instagram.com/openbetaproject/)
+
+# Step 2 Deep Dive About Open Beta
+
+- [ ] Read [our roadmap](https://docs.openbeta.io/introduction/roadmap)
+- [ ] Read [our blog]()
+
+# Step 3 Organization Workflow
+Our workflow involves both unsynchronized communications through GitHub issues, pull requests (PRs), and a Discord channel, as well as regular synchronized meetings. Here's an overview of our organization workflow:
+
+## Unsynchronized Communication
+
+### GitHub Issues:
+
+- Use GitHub issues to report bugs, suggest features, or discuss project-related topics.
+- If you'd like to work on the issue, assign yourself.
+- If you are unsure about how to start, you could @ our core dev team members(list here). Alternatively, you could post in our discord channel to get help.
+
+### Pull Requests (PRs):
+
+- Once you know how to start working on the issue, you could fork the project repository and create a branch for your changes. 
+- Submit PRs to propose changes, bug fixes, or new features.
+- Request reviews from our core dev team members, engage in PR discussions, respond to feedback, and iterate on the changes based on reviewer comments.
```

Comment:
I would add guidance on who to ask for a review.  It can be intimidating and confusing for a first time contributor to decide who to ask.  Worst case scenario, github's suggested reviewers are disengaged from the and a new contributor's PR languishes forever.  For example,

> If you're not sure who should review your changes, ask @.... or @... and they will direct PR to an appropriate reviewer.

Maybe @actuallyyun would be a good person for now, as the new dev onboarder.

Reply:
Great idea!